packages/terra-application/docs/README.md

```diff
@@ -0,0 +1,285 @@
+# Terra Application
+
+The `terra-application` package provides a baseline of functionality for applications built with Terra components.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application`
+
+## Usage
+
+The `terra-application` package should be used both by applications and libraries to define a baseline of expected features and functionality for their components.
+
+For application packages:
+
+1. `terra-application` must be specified as a dependency.
+2. A single `<Application />` component must be rendered as high as possible in the component tree.
```

Comment:
nit: We use the following verbiage on the i18n getting started page in Terra UI. `This component should be your root component and wrap all components in your app. This is to help ensure that the entire application is within the same configured i18n context provided by react-intl's <IntlProvider />.` 

It might make sense to copy/paste and reword that for the second bullet point here and remove the react-intl part, but note that this is needed for context provider concerns.

Something like.

```
This component should be your root component and wrap all components in your app. This is to help ensure that the entire React application is within the same configured React context providers. 
```

Reply:
I've expanded this section pretty heavily. Please take a look and see what you think.

Reply:
LGTM